driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -169,6 +169,7 @@ trait Io {
         result_folder: &str,
         solver_type: SolverType,
         time_limit: Duration,
+        min_avg_fee_per_order: u128,
```

Comment:
Since we have access to self in this method, do we need the extra argument? Can't we just use self. min_avg_fee_per_order inside the method?

Reply:
self is referring to the DefaultIo or Io,  but not to the optimizaiton_price_finder itself